imap_processing/hit/l1b/hit_l1b.py

```diff
@@ -47,6 +49,7 @@ def hit_l1b(l1a_dataset: xr.Dataset):
     logger.info("Creating CDF files for HIT L1B data")
     cdf_filepaths = []
     for dataset in datasets:
+        dataset.attrs["Data_version"] = data_version
```

Comment:
Does this also need to make this call? `attr_mgr.add_global_attribute("Data_version", data_version)`

Reply:
I asked Maxine about this and her explaining how these should be used reminded me that HIT hasn't changed to using YAML, so that's why 

```python
dataset.attrs["Data_version"] = data_version
```
is being used instead of
```python
attr_mgr.add_global_attribute("Data_version", data_version)
```

This comment can be ignored for now.